subsys/usb/usb_device.c

```diff
@@ -332,6 +332,10 @@ static void usb_handle_control_transfer(u8_t ep,
 
 		/* Send smallest of requested and offered length */
 		usb_dev.data_buf_residue = MIN(usb_dev.data_buf_len, length);
+		if (usb_dev.data_buf_len > CONFIG_USB_REQUEST_BUFFER_SIZE) {
+			LOG_ERR("REQUEST_BUFFER too small");
+		}
```

Comment:
Can we add a call to `k_oops()` here so that the error is unconditionally detected

Reply:
done